hyperspy/samfire_utils/samfire_worker.py

```diff
@@ -23,9 +23,6 @@
 from queue import Empty
 import dill
 import numpy as np
-import matplotlib
-
-matplotlib.rcParams['backend'] = 'Agg'
 
```

Comment:
@to266, I removed this when I tidied up the way the matplotlib backend is set. Could you please confirm that there is no particular reason to have this here?